src/App.js

```diff
@@ -1,25 +1,37 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from 'react';
+import { Route, Routes } from 'react-router-dom';
+import Calculator from './components/Calculator';
+import Navbar from './components/Navigation';
+import Home from './components/pages/Home';
+import Quote from './components/pages/Quote';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="main-app">
 
+        <Navbar />
+        <Routes>
+          <Route
+            path="/math-magicians/"
+            element={(<Home />)}
+          />
```

Comment:
- [ ] Kindly make sure to set the `home` page to be the default page which will be rendered once we start the application. You can do that by setting the `path` to `/`.